files/es/web/css/border/index.html

```diff
@@ -51,7 +51,7 @@ <h3 id="Valores" name="Valores">Valores</h3>
 
 <h3 id="Ejemplos" name="Ejemplos">Ejemplos</h3>
 
-<p><a href="/samples/cssref/border.html">Ver El Ejemplo Vivo</a></p>
+<p><a href="/samples/cssref/border.html">Ver Ejemplo</a></p>
```

Comment:
En este caso podria mantenerse como `Ver ejemplo en vivo`.

```suggestion
<p><a href="/samples/cssref/border.html">Ver ejemplo en vivo</a></p>
```